index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+
+    <!-- Navigation Header -->
+
+    <header class="header">
+        <nav class="navbar">
+            <h4 class="header-name">Augustine Abah</h4>
+            <img src="images/Union.png" alt="">
+        </nav>
+    </header>
+
+    <!-- Introduction -->
+    <img src="images/Ellipse18.svg" class="introduction-image" alt="">
+    <section>
+        <div class="introduction">
+            <h1 class="introduction-name">Augustine Abah</h1>
+            <h2 class="soft-developer">Software Developer</h2>
+            <p class="intro-paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
+            <img src="images/Devider.png" class="intro-line" alt="">
+        </div>
+        
+    </section>
+
+    <!-- Experience -->
+
+    <img src="images/Ellipse19.svg" class="experience-img" alt="">
```

Comment:
- [ ] Please use proper background images for this section. You can so that by using the CSS background-image property.